Backpack/HorizontalNavigation/Classes/BPKHorizontalNavigation.m

```diff
@@ -108,7 +108,9 @@ - (void)updateBarAppearance {
         if (UIAccessibilityIsReduceMotionEnabled()) {
             animationDuration = 0.0;
         }
-
+        
+        [self setNeedsLayout];
+        [self layoutIfNeeded];
```

Comment:
There might be pending changes in the layout that we need to do before executing the animation, otherwise these pending changes will be animated as well.

Reply:
I think that here you only need the `layoutIfNeeded` that's the synchronous call that tells the system you want a layout and redraw of a view and its subviews, and also you want it done immediately without waiting for the update cycle.
Have you try it this solution without the `setNeedsLayout` here?

Reply:
You are absolutely right. Just deleted `setNeedsLayout` and it works fine.